main.js

```diff
@@ -69,6 +91,12 @@ ideaSection.addEventListener('click', function(event){
           ideas[i].isFavorited = true
         } else if (ideas[i].isFavorited === true) {
           ideas[i].isFavorited = false
+          ideas[i].isPushed = false
```

Comment:
This function could use some refactoring. Could use truthy/falsey values in some of the if statements